### PR TITLE
Add audio set position to the database

### PR DESCRIPTION
**Pull request by obulat**
## Fixes

Related to https://github.com/WordPress/openverse-api/issues/157#issuecomment-911539017 by @dhruvkb

## Description

This PR completes the implementation of set position column for Audio from #161 by adding the set_position column to the  audio database.

## Technical details

The work on this PR is based on the database loader refactoring in #168. 

## Tests

To test that everything works correctly, first make sure that your old database is deleted. 

_I'm not sure if it's due to the fact that the database was open in DataGrip, or something else, but when I ran `just down -v` after updating the `0006_openledger_audio_schema.sql` script, and started the containers again with `just up`, the table structure didn't change. Which means, the loading errorred because the audio table did not have `set_position` column. What I did in the end was deleting running `just down -v` AND deleting the postgres container._

Run the containers, log in to Airflow Web UI. Un-pause `jamendo`, `tsv_to_postgres_loader` and `recreate_audio_popularity_calculation` dags. Jamendo task runs for a very long time, so it's best to check in the logs that some lines have been written to the tsv file:

Screen Shot 2021-09-10 at 4 02 20 PM

 and mark the `pull_image_data` as success:
Screen Shot 2021-09-10 at 3 59 08 PM

A couple of minutes later, the `tsv_to_postgres_loader` will load the tsv to postgres db. Then you can trigger the `recreate_audio_popularity_calculation` dag to create the audio_view which is the data view that is used by the API.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [ ] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [x] I added or updated tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by sarayourfriend** [comment]
What does `set_position` represent?

**Comment by krysal** [comment]
@sarayourfriend It's the position of the audio in its associated set (if exists). For example, the number of an episode of a podcast series, or just the audio position in a music album.

File: openverse_catalog/dags/common/storage/audio.py [hunk 1]
```diff
@@ -76,10 +76,14 @@
         ),
     ),
     columns.JSONColumn(
-        # set name, set thumbnail, position of audio in set, set url
+        # set name, thumbnail, url, identifier etc.
         name="audio_set",
         required=False,
     ),
+    columns.IntegerColumn(
+        name="set_position",
```
Comment (zackkrida) [inline openverse_catalog/dags/common/storage/audio.py:84]:
```suggestion
    columns.IntegerColumn(
    	# The position of the audio within an audio set, for example:
    	# the 200th episode of a podcast or the 2nd track on an album.
        name="set_position",
```

**Review by zackkrida — APPROVED** [review]
Thank you for the detailed testing instructions @obulat !

**Review by krysal — APPROVED** [review]
Impressive! Works great, and seems like a breeze to add new columns now, compared to before 😄

Interestingly, I got some negative values in the `set_position` column.

**Comment by zackkrida** [comment]
@obulat LGTM!

**Comment by obulat** [comment]
Closing in favor of #196

**Pull request closed without merging**